jetstream_pt/ray_worker.py

```diff
@@ -353,7 +352,7 @@ def _call_model_generate(
     args = (tokens, input_pos, caches_obj, mask)
     paramst, argst = torchjax.to_torch((weights, args))
     with self._lock:
-      with torchjax.jax_mode():
```

Comment:
Actually there is a typo here, should not have ()

Reply:
Yes, it's replace with  torch_xla2.default_env() now.